### PR TITLE
Small UI fixes

### LINKED ISSUE #12: Align suggestion menu position with replace menu
### What should we do?

Align position with change element menu
image

### Why should we do it?

To have seamless UI interactions

### LINKED ISSUE #20: Context pad entry not shown as active when popup open
### Describe the Bug

When opening the popup the context pad entry is not shown as active:

image

Compare to:

image

### Environment

- Library version:  0.2.0

### PR DESCRIPTION
**Pull request by marstamm**
This PR:
 - aligns positioning of the popup menu with the replace menu #12
 - marks the entry as active while the menu is open #20

Recording 2024-04-04 at 11 30 24

closes #12
closes #20

### CODE REVIEW

File: lib/context-pad/RefactoringsContextPadProvider.js [hunk 1]
```diff
@@ -34,15 +36,22 @@ export default class RefactoringsContextPadProvider {
...
-          action: () => {
-            const pad = this._contextPad.getPad(element);
+        refactorings: {
+          action: {
+            click: (event) => {
+              const pad = this._contextPad.getPad(element);
+              this._openPopupMenu(pad, element);
 
-            this._openPopupMenu(pad, element);
+              const entry = domClosest(event.delegateTarget || event.target, '.entry', true);
```
Comment (philippfromme) [inline lib/context-pad/RefactoringsContextPadProvider.js:49]: 🙈 Isn't this a hack? How does the replace popup menu to this? Active states should be a feature of the context pad.

Comment (marstamm) [reply lib/context-pad/RefactoringsContextPadProvider.js:49]: The method is consistent with how the new context pad handles active state:
https://github.com/camunda/improved-canvas/blob/7dc5052c7057f6ea1e5f7b08bd62a12e8a885e1e/lib/common/contextPad/ImprovedContextPadProvider.js#L110-L115

The "get the entry dom node from event" is consistent with the PopupMenu Code:
https://github.com/bpmn-io/diagram-js/blob/cc602c41a5b05510de4b81a753096919e47bc8b4/lib/features/popup-menu/PopupMenu.js#L585

Comment (marstamm) [reply lib/context-pad/RefactoringsContextPadProvider.js:49]: I don't think the context pad handles open state, as we never had this feature in the old context pad

**Review by philippfromme — APPROVED** [review]
✅

**Pull request merged**